homework/02LessonTwo.js

```diff
@@ -26,6 +26,11 @@ var alwaysTrue = function () {
 // argument1 is equal to argument2
 
 function equals(argument1, argument2) {
+  if (argument1 === argument2) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
good! now, can you think of a way to write the function without any `if` / `else` statements?